website/www/site/content/en/documentation/io/built-in/snowflake.md

```diff
@@ -152,305 +143,6 @@ Snowflake IO library supports following options that can be passed via the [comm
 
 `--portNumber` Port number. Optional.
 
-`--loginTimeout` Login timeout. Optional.
```

Comment:
I am noyt sure about removing `loginTimeout`. @DariuszAniszewski is it something for authentication?

Reply:
It should stay - it's available in `SnowflakeBasicDataSource` and we can set it.

Reply:
re-added